command/loadbot/loadbot_command.go

```diff
@@ -0,0 +1,213 @@
+	return "loadbot"
+}
+
+func (l *LoadbotCommand) Synopsis() string {
+	return l.GetHelperText()
+}
+
+func (l *LoadbotCommand) Help() string {
+	l.Meta.DefineFlags()
+	l.DefineFlags()
+
+	return helper.GenerateHelp(l.Synopsis(), helper.GenerateUsage(l.GetBaseCommand(), l.FlagMap), l.FlagMap)
+}
+
+func (l *LoadbotCommand) Run(args []string) int {
+	flags := flag.NewFlagSet(l.GetBaseCommand(), flag.ContinueOnError)
+
+	var tps uint64
+	var accountsRaw helperFlags.ArrayFlags
+	var gasLimit uint64
+	var gasPriceRaw string
+	var urls helperFlags.ArrayFlags
+	var chainID uint64
+	var count uint64
+	var valueRaw string
+	var gRPC string
+
+	flags.Uint64Var(&tps, "tps", 100, "")
+	flags.Var(&accountsRaw, "account", "")
+	flags.Uint64Var(&gasLimit, "gasLimit", 1000000, "")
+	flags.StringVar(&gasPriceRaw, "gasPrice", "0x100000", "")
+	flags.Var(&urls, "url", "")
+	flags.Uint64Var(&chainID, "chain-id", helper.DefaultChainID, "")
+	flags.Uint64Var(&count, "count", 1000, "")
+	flags.StringVar(&valueRaw, "value", "", "")
+	flags.StringVar(&gRPC, "grpc", "", "")
+
+	if err := flags.Parse(args); err != nil {
+		l.UI.Error(fmt.Sprintf("failed to parse args: %v", err))
+		return 1
+	}
+
+	// Parse accountsRaw
+	var addresses = []types.Address{}
+	if accountsRaw == nil {
+		l.UI.Error("failed to parse accounts provided that will be used by the loadbot")
+		return 1
+	}
+	for _, account := range accountsRaw {
+		placeholder := types.Address{}
+		if err := placeholder.UnmarshalText([]byte(account)); err != nil {
+			l.UI.Error(fmt.Sprintf("Failed to decode account address: %v", err))
+			return 1
+		}
+		addresses = append(addresses, placeholder)
+	}
+
+	// Parse urls
+	if len(urls) == 0 {
+		l.UI.Error("please provide at least one node url to run the loadbot")
+		return 1
+	}
+
+	value, err := types.ParseUint256orHex(&valueRaw)
+	if err != nil {
+		l.UI.Error(fmt.Sprintf("Failed to decode to value: %v", err))
+		return 1
+	}
+	gasPrice, err := types.ParseUint256orHex(&gasPriceRaw)
+	if err != nil {
+		l.UI.Error(fmt.Sprintf("Failed to decode to gasPrice: %v", err))
+		return 1
+	}
+
+	var accounts []types.Address
+	for _, account := range accountsRaw {
+		acc := types.Address{}
+		if err := acc.UnmarshalText([]byte(account)); err != nil {
+			l.UI.Error(fmt.Sprintf("Failed to decode to address: %v", err))
+			return 1
+		}
+
+		accounts = append(accounts, acc)
+	}
+
+	metrics, err := Execute(&Configuration{
+		TPS:       tps,
+		Value:     value,
+		Gas:       gasLimit,
+		GasPrice:  gasPrice,
+		Accounts:  accounts,
+		RPCURLs:   urls,
+		ChainID:   chainID,
+		TxnToSend: count,
+		GRPCUrl:   gRPC,
+	})
+	if err != nil {
+		l.UI.Error(fmt.Sprintf("failed to execute loadbot: %v", err))
+		return 1
+	}
+
+	l.UI.Info("Loadbot execution finished. Got following metrics :")
+	l.UI.Info(fmt.Sprintf("Transactions submitted: %v", metrics.Total))
+	l.UI.Info(fmt.Sprintf("Transactions failed: %v", metrics.Failed))
+	l.UI.Info(fmt.Sprintf("Duration: %v", metrics.Duration))
```

Comment:
Consider packing this into a KV output, like we do for example in the ibft-init command

Reply:
Added in https://github.com/0xPolygon/polygon-sdk/pull/197/commits/45b67ce0874d8dbd2120ee3c409098b5a1f99c15